test/karma.conf.js

```diff
@@ -151,21 +151,21 @@ function getCustomLaunchers(){
       base: 'BrowserStack',
       browser: 'chrome',
       os: 'Windows',
-      os_version: '8.1'
+      os_version: '10'
     },
 
     firefox_bs: {
       base: 'BrowserStack',
       browser: 'firefox',
       os: 'Windows',
-      os_version: '8.1'
+      os_version: '10'
     },
 
     safari_bs: {
       base: 'BrowserStack',
       browser: 'safari',
       os: 'OS X',
-      os_version: 'El Capitan'
```

Comment:
we may want to keep this as el capitan so that it tests against safari 9 so we don't break it. We could add high sierra as a separate test run as well, though.